packages/graphql-playground-react/src/components/Playground/GraphQLEditor.tsx

```diff
@@ -757,14 +759,19 @@ export class GraphQLEditor extends React.PureComponent<
   // Private methods
 
   public reloadSchema = async () => {
-    const result = await this.props.schemaFetcher.refetch(
-      this.props.session.endpoint || this.props.endpoint,
-      this.convertHeaders(this.props.session.headers),
-    )
-    if (result) {
-      const { schema } = result
-      this.setState({ schema })
-      this.renewStacks(schema)
+    try {
+      this.setState({ isReloadingSchema: true })
+      const result = await this.props.schemaFetcher.refetch(
+        this.props.session.endpoint || this.props.endpoint,
+        this.convertHeaders(this.props.session.headers),
+      )
+      if (result) {
+        const { schema } = result
+        this.setState({ schema, isReloadingSchema: false })
+        this.renewStacks(schema)
+      }
+    } catch (e) {
+      this.setState({ isReloadingSchema: false })
```

Comment:
What about showing some feedback when reloading the schema fails?

> How about displaying a little notification dot (similar how the Twitter feed does it) next to the reload icon which is there until you change/run a query.

Something like a red dot? 🔴 

With maybe a tooltip saying "invalid schema"?

Or reusing the "unreachable schema" state?

Reply:
Yes, we can reuse that state for this purpose.